scripts/find-mem-leak.cocci

```diff
@@ -2,8 +2,8 @@
 expression x, y;
 statement S;
 @@
-  x = mbedtls_malloc(...);
-  y = mbedtls_malloc(...);
+  x = mbedtls_calloc(...);
+  y = mbedtls_calloc(...);
   ...
```

Comment:
@hanno-arm: Indeed, some of these scripts do flag false positives. As far as I can tell, we are not executing these scripts from any other test scripts.